components/serializer.rst

```diff
@@ -757,7 +757,7 @@ When serializing, you can set a callback to format a specific object property::
 
     // all callback parameters are optional (you can omit the ones you don't use)
     $dateCallback = function (object $innerObject, object $outerObject, string $attributeName, ?string $format = null, array $context = []): string {
-        return $innerObject instanceof \DateTime ? $innerObject->format(\DateTime::ISO8601) : '';
+        return $innerObject instanceof \DateTime ? $innerObject->format(\DateTime::ATOM) : '';
```

Comment:
Curious if we should use the constant from \DateTimeInterface 🤔

Reply:
We could ... but since we're using `instanceof \DateTime` in the same line of code, I'd say using `\DateTime::ATOM` feels more natural here. Thanks.